eq-author/src/components/RichTextEditor/index.js

```diff
@@ -348,21 +378,79 @@ class RichTextEditor extends React.Component {
       : this.hasInlineStyle(editorState, style);
   };
 
+  state = { show: false, text: "", multiline: false };
+
   handlePaste = (text) => {
-    this.handleChange(
-      EditorState.push(
-        this.state.editorState,
-        Modifier.replaceText(
-          this.state.editorState.getCurrentContent(),
-          this.state.editorState.getSelection(),
-          text.replace(/\n/g, " ")
+    if (/\s{2,}/g.test(text)) {
+      this.setState({
+        show: true,
+        multiline: false,
+        text: text,
+      });
+    } else {
+      this.handleChange(
+        EditorState.push(
+          this.state.editorState,
+          Modifier.replaceText(
+            this.state.editorState.getCurrentContent(),
+            this.state.editorState.getSelection(),
+            text.replace(/\n/g, " ")
+          )
         )
-      )
-    );
+      );
+    }
 
     return "handled";
   };
 
+  handlePasteMultiline = (text) => {
+    if (/\s{2,}/g.test(text)) {
+      this.setState({
+        show: true,
+        multiline: true,
+        text: text,
+      });
+      return "handled";
+    } else {
+      return true;
+    }
+  };
```

Comment:
This function can return two different types (string "handled" and Boolean `true`)

This can be changed to return only one type, either returning "handled" and "not-handled" strings, or `true` and `false` Boolean values.

Reply:
Changed 'true' to "not-handled"